pkgs/by-name/ke/keep-sorted/package.nix

```diff
@@ -0,0 +1,35 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+}:
+
+buildGoModule rec {
+  pname = "keep-sorted";
+  version = "0.2.0";
+
+  src = fetchFromGitHub {
+    owner = "google";
+    repo = "keep-sorted";
+    rev = "v${version}";
+    hash = "sha256-bCV0XcwgyFTORl/RF1BS7vsM8DmU0Wox3OIEuZBrwSs=";
+  };
+
+  vendorHash = "sha256-yaeqfMAJbQdrqZ0uco6Y5T8vnfjlBJY4IQuGzZg3Ubw=";
+
+  CGO_ENABLED = "0";
+
+  ldfags = [ "-s" "-w" ];
+
+  checkFlags = [
+    # Test tries to find files using git
+    "-skip=^TestGoldens"
+  ];
+
+  meta = {
```

Comment:
```suggestion
  meta = {
    changelog = "https://github.com/google/keep-sorted/releases/tag/v${version}";
```

Reply:
Not sure about this one, they don't do GitHub releases, so this will just point to the bare tag and won't be that helpful: https://github.com/google/keep-sorted/releases/tag/v0.2.0
But I can add it anyway if you like.

Reply:
Oh, and in an ideal world tags would be recognized as releases, but here we are with GitHub and their lock-in strategy.